### PR TITLE
Chris work

### PR DESCRIPTION
**Pull request by chris-hinson**
merge me bitch x 2

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,51 +1,49 @@
...
 }
 
 #[derive(Component)]
-struct SlideDeck{
+struct SlideDeck {
     total_slides: usize,
     current_slide: usize,
 }
 
+//gavin is ugly
```
Comment (Elsklivet) [inline src/main.rs:14]: ???

**Review by Elsklivet — CHANGES_REQUESTED** [review]

File: src/main.rs (hunk 1 above)
Comment (chris-hinson) [reply src/main.rs:14]: adds important and nessecary context

**Comment by Elsklivet** [comment]
> merge me bitch x 2

not very nice

**Comment by chris-hinson** [comment]
ok youre right. im sorry.
*please* merge me bitch x2

**Comment by Elsklivet** [comment]
Your credit slide is already merged i am not merging gavin is ugly

**Pull request closed without merging**